saleor/graphql/webhook/mutations/webhook_dry_run.py

```diff
@@ -102,10 +104,18 @@ def validate_input(cls, info, **data):
         cls.validate_event_type(event_type, object_id)
         cls.validate_permissions(info, event_type)
 
-        object = cls.get_node_or_error(info, object_id, field="objectId")
+        object = cls.get_instance(info, object_id)
 
         return event_type, object, query
 
+    @classmethod
+    def get_instance(cls, info: ResolveInfo, object_id):
+        type, _id = from_global_id_or_error(object_id, raise_error=False)
+        if type == "Sale":
+            object_id = cls.get_global_id_or_error(object_id, "Sale")
+            return discount_models.Promotion.objects.get(old_sale_id=object_id)
+        return cls.get_node_or_error(info, object_id, field="objectId")
```

Comment:
We need to do the same for `webhookTrigger` mutation.

Reply:
Done